Needlework.Net.Desktop/Views/MainWindow.axaml

```diff
@@ -82,6 +82,7 @@
 					<ItemsRepeater.ItemTemplate>
 						<DataTemplate>
 							<Border Margin="4">
+								<!-- TODO add some background to InfoBar (currently it is (probably) transparent and you can see other things under it) --> 
```

Comment:
It's a known upstream problem with FluentAvalonia. I might use a theme color that isn't transparent.

Reply:
I didn't know that, should I delete the comment (or edit it to say it's a known issue and add a link to the issue)?